index.rst

```diff
@@ -4,9 +4,173 @@ Collimated Beam Projector Installation and ComCam Testing
 
 .. abstract::
 
-   Report on the initial installation and testing of the laser and Collimated Beam Projector.
+   Report on the initial installation and testing of the laser and Collimated Beam Projector during the ComCam campaign.
 
-Add content here
-================
+Introduction
+============
 
-See the `Documenteer documentation <https://documenteer.lsst.io/technotes/index.html>`_ for tips on how to write and configure your new technote.
+The Collimated Beam Projector (CBP) is a Schmidt Reflector run in reverse. We use an Ekspla tunable laser connected to an integrating sphere to illuminate the CBP focal plane. We design custom masks that we place at the focal plane and then CBP sends out collimated beams of light that can be reimaged by ComCam. The beamsize is 240 mm, so we illuminate only part of the Simonyi Telescope mirror. However, by doing a copointing dance, we can illuminate any part of the mirror and any point on the focal plane. A NIST calibrated photodiode in the integrating sphere allows us to know exactly how much light we are sending into the telescope, allowing us to perform very precise photometry of our spots.
+
+The main goal of the CBP is to measure the throughput of collimated light with high precision, but more generally with the CBP we can:
```

Comment:
The primary use of the CBP will be to measure the wavelength-dependent throughput of the Rubin telescope and LSSTCam with high precision. Additionally, it will allow us to do the following: